_licenses/cecill-2.1.txt

```diff
@@ -0,0 +1,580 @@
+---
+title: CeCILL Free Software License Agreement v2.1
+spdx-id: CECILL-2.1
+source: http://www.cecill.info/licences/Licence_CeCILL_V2.1-fr.txt
+
+description: French open-source license made by the CEA, INRIA and CNRS, compatible with the GNU GPL v2.0 and v3.0
+
+how: Create a text file (typically named LICENSE or LICENCE) in the root of your source code and copy the text of the license into the file. 
+
+using:
```

Comment:
BMC-Tools is the only one of these that would be recognized by licensee -- single license file in the repository root with the canonical license text only. Are there other examples like that?